dotCMS/src/main/java/com/dotcms/business/bytebuddy/ByteBuddyFactory.java

```diff
@@ -0,0 +1,161 @@
+package com.dotcms.business.bytebuddy;
+
+import com.dotcms.business.CloseDB;
+import com.dotcms.business.CloseDBIfOpened;
+import com.dotcms.business.WrapInTransaction;
+import com.dotcms.test.TransactionalTester;
+import com.dotcms.util.LogTime;
+import com.dotcms.util.VoidDelegate;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.util.Logger;
+import net.bytebuddy.agent.ByteBuddyAgent;
+import net.bytebuddy.agent.builder.AgentBuilder;
+import net.bytebuddy.asm.Advice;
+import net.bytebuddy.description.annotation.AnnotationDescription;
+import net.bytebuddy.description.method.MethodDescription;
+import net.bytebuddy.dynamic.ClassFileLocator;
+import net.bytebuddy.matcher.ElementMatchers;
+import net.bytebuddy.utility.JavaModule;
+import org.apache.logging.log4j.Level;
+
+import java.lang.instrument.Instrumentation;
+
+import static net.bytebuddy.matcher.ElementMatchers.*;
+
+public class ByteBuddyFactory {
+
+    private static final ByteBuddyFactory INSTANCE = new ByteBuddyFactory();
+
+
+
+
+    private ByteBuddyFactory() {
+        init();
+    }
+
+    private void init() {
+        try {
+            premain(null, ByteBuddyAgent.install());
+            Logger.info(ByteBuddyFactory.class, "Loaded ByteBuddy Advice");
+
+        } catch (Exception e) {
+            Logger.error(ByteBuddyFactory.class, "Cannot install ByteBuddy Advice");
```

Comment:
I would log the exception such as
Logger.debug(

Reply:
Why debug?,  if this happens it is a fatal error and the system will fail so should be an error.